src/components/Geolocation/Geolocation.js

```diff
@@ -1,37 +1,113 @@
+import './geolocation.scss';
+
 export class Geolocation extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      showGeo: true,
       latitude: 0,
-      longitude: 0
+      longitude: 0,
+      userLatitude: 0,
+      userLongitude: 0,
```

Comment:
hmmm, I'm not sure you need userLatitude and userLongitude... it is an excess logic